main.py

```diff
@@ -287,7 +286,13 @@ def log_bad_request(request):
 
 @app.route('/')
 def index(marker=None, message=None):
-    context = {'minimal_zoom': MINIMAL_ZOOM, 'url': request.base_url, 'index_url': request.url_root}
+    context = {'minimal_zoom': MINIMAL_ZOOM,
```

Comment:
A constants class sounds like a good idea. The members' names can then be identical to the ones you send (lower case with underscore).